contracts/Comptroller/Diamond/facets/PolicyFacet.sol

```diff
@@ -248,7 +267,14 @@ contract PolicyFacet is XVSRewardsHelper {
         address borrower,
         uint actualRepayAmount,
         uint seizeTokens
-    ) external {}
+    ) external {
+        if (address(prime) != address(0)) {
```

Comment:
`VToken.seize` is invoked during a liquidation in `VToken.liquidateBorrowFresh`, before invoking `liquidateBorrowVerify`. So, `seizeVerify` will be invoked too. It seems we are going to invoke the `updateScore` for the vtoken collateral two times during the liquidation.